acceptance/lib/helper.rb

```diff
@@ -167,12 +167,7 @@ def upgrade_package(host, name)
   end
 
   def get_defaults_var(host, varname)
-    if master.is_pe?
```

Comment:
I'm actually okay with this and agree that this definitely feels better than proliferating conditional logic across tests - although presumably we could have also done this by just putting the conditional check in a helper function that would be called from the different tests instead of defining a new `option`.

I wasn't sure if `is_pe?` is some sort of "standard" recommended for this kind of differentiation in the Beaker world.  In any event, I'm happy with what is on the PR for this right now and am not suggesting we change it in the future.